ifrs17-template/Test/ReimportTest.ipynb

```diff
@@ -0,0 +1,291 @@
+                "\n",
+                "\nvar dt11State = new DataNodeState { DataNode = \"DT1.1\", State = State.Active, Year = args.Year, Month = args.Month, Partition = partitionByReportingNode.Id };",
+                "\n",
+                "\nvar dtr11State = new DataNodeState { DataNode = \"DTR1.1\", State = State.Active, Year = args.Year, Month = args.Month, Partition = partitionByReportingNode.Id };",
+                "\n",
+                "\nvar dt13State = new DataNodeState { DataNode = \"DT1.3\", State = State.Active, Year = args.Year, Month = args.Month, Partition = partitionByReportingNode.Id };"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Test Transactional Data"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Reimport Openings for DataNode at Inception"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "Create openings for a data node at inception (DT1.1) and import more than once."
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var openings = @\"",
+                "\n@@Main,,,,",
+                "\nReportingNode,Year,Month,,",
+                "\nCH,2021,3,,",
+                "\n@@Opening,,,,",
+                "\nDataNode,EstimateType,AmountType,AccidentYear,Value",
+                "\nDT1.1,C,,,72.2",
+                "\nDT1.1,AA,PR,,-1.5",
+                "\nDT1.1,OA,PR,,1.5",
+                "\n\";"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var activity = await ReimportAndCompare((openings, ImportFormats.Opening), new [] { dt11State, dtr11State });",
+                "\nactivity"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "activity.Status.Should().Be(ActivityLogStatus.Succeeded);"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Reimport Actuals for Both: DataNode at Inception and DataNode and Not at Inception"
```

Comment:
## Reimport Actuals for Both: DataNode at Inception and DataNode and Not at Inception =>
## Reimport Actuals for Both: DataNode at Inception and DataNode Not at Inception